docs/static/monitoring/monitoring-prereq-setup-es.asciidoc

```diff
@@ -1,5 +1,5 @@
 //[[set-up-monitoring]]
 //=== Set up {es} monitoring (stack monitoring only)
 
-Set up {ref}/monitoring-overview.html[{es} monitoring].
+With *Stack Monitoring* it is recommended to set up {ref}/monitoring-overview.html[{es} monitoring] together. Otherwise, {ls} becomes a Standalone Cluster.
```

Comment:
```suggestion
To bind {ls} metrics to an {es} cluster, also set up {ref}/monitoring-overview.html[{es} monitoring].
```

Reply:
Agreed, I have seen multiple standalone issues under KB repo and better not to mention for now.